src/vm/classnames.h

```diff
@@ -93,19 +93,13 @@
 #define g_ExceptionClassName "System.Exception"
 #define g_ExecutionEngineExceptionClassName "System.ExecutionEngineException"
 
-#define g_MarshalByRefObjectClassName "System.MarshalByRefObject"
```

Comment:
I think we may have a CoreFX test for this?

Reply:
Yes, this MarshalByRefObject exists in public surface. It is just a dummy type for backward compatibility, not treated in any special way by the runtime.